### ISSUE TITLE
Felicity has a troublesome temperature sensor

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1203]
Felicity is reporting problems with her temperature sensor:

> ERROR - STOPPED
> Temp sensor
> Go to:
> ultimaker.com/ER01

That [error page](https://ultimaker.com/en/resources/19484-stopped-temp-sensor) has a list of troubleshooting steps to run through.  I haven't tried any of them yet.

**Comment by jackie1050** [comment]
@cheapjack and I ran through the troubleshooting steps above. 
We took the cover off the main board and checked that the cable was connected to 'TEMP1' and that the red end wires were properly inserted in the connector - they were.
We then swapped the PT100 sensor (TEMP1) with the one of the heated bed (TEMP3) where they connect to the main board and tried to heat the nozzle to 90C. This led to ER03, which is a heating error (trying to heat but nothing happening).
We swapped back the sensors and left the printer.

We need to reswap the sensors and heat the bed. If we get ER01 then the sensor is faulty.

**Comment by jackie1050** [comment]
I swapped the sensors again and heated up the bed - the screen continued to give the actual temperature as 23C, which is correct as the nozzle was at room temperature - so it didn't show the error we were expecting and we need to do more diagnostics.
THE SENSORS ARE STILL SWAPPED.

**Comment by Thingomy** [comment]
Thermistor confirmed bad; I have stripped down the head and removed the offending article. Head of printer is currently in pieces and in need of a replacement thermistor. Threading the wire for replacement down wiring loom will be a significant task.

Note that one of the fans has a crushed and shorted wire, this fan will need replacing too.

**Comment by Thingomy** [comment]
Thermocouple: https://www.makers-hut.com/product/ultimaker-2-um2-pt100b-sensor-thermocouple-hotend-temperature-sensor/ £24.99
Fan: https://www.makershop3d.com/ultimaker-spare-parts/676-um2-ext-go-side-fan.html 10,90 €

**Comment by Thingomy** [comment]
Wire colours: 
centre fan - +ve blue, -ve orange
side fans (in series) - +ve yellow, -ve green

red and brown - not connected at either end

**Comment by Sean-anotherone** [comment]
PT100 sensors are cheap !   
https://www.ebay.co.uk/itm/PT100-Sensor-Probe-Platinum-Chip-EN-60751-5-Pieces/291800885504?hash=item43f0b02900:g:U7QAAOSwajVUSBXV

Would need a bit of hacking to fit it into the existing can, but rather better value than £25 !

A tenner for a 30mm? fan is way OTT too, : 2 in series might be 5v fans running on a 12v supply?

https://www.ebay.co.uk/itm/5V-mini-3cm-30mm-30x30x10mm-2pin-pont-brushless-cooling-cooler-fan-3010s-In-DY3/372683918607?hash=item56c5b14d0f:g:fDgAAOSwVcZcHGA5

Just me being a cheapskate, sorry, economically conscious.  :)

**Comment by jackie1050** [comment]
@DoESsean - if @Sean-anotherone prepared to do any required mods, then worth trying

**Comment by magman2112** [comment]
Hi - Sean here, just using Magmans laptop....

I took a look at the broken bits that are available and a repair isn't really feasible, I would suggest buying the part that has already been identified and taking some measurements before fitting it, then matching up a cheaper part using those measurements so a discount repair can be made next time.

**Comment by DoESsean** [comment]
I've ordered the items @Sean-anotherone listed as cheaper alternatives. If they don't work out, I'll chalk it up to experience and go for the name brand stuff next time.

**Comment by jackie1050** [comment]
If you read the last post, he'd changed his mind, but I think it was a close decision so hopefully worth trying, as you say...

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by DoESsean** [comment]
Ah drat...

**Comment by amcewen** [comment]
The fans have arrived (although it looks like they've got different connectors to the ones on the printer, so we'll need to splice the wires instead and use the connector from the old fan), as have the PT100 sensors.

They're going to be near impossible to switch out, so I think we should just order one of the Ultimaker replacements.  The one @Thingomy found is out-of-stock, so I've ordered one of these https://www.makershop3d.com/ultimaker-spare-parts/24-temperature-sensor-pt100-b-ultimaker.html

**Comment by amcewen** [comment]
A reminder that we should put one of the new PTFE couplers (from #1191) in when Felicity's hot end is reassembled.

**Comment by Sean-anotherone** [comment]
Alas, the repair did not last...
The coupling spring that maintains the pressure on the teflon piece has been lost and while a <broadly similar> replacement spring was fitted, this did not apply enough pressure to prevent the coupler lifting and the machine getting bunged up.

I did another teardown and found the metal coupler blocked solid, when clearing out that blockage, it cracked.
To get Felicity up & running some parts are required : 

1. A replacement metal coupler piece for the print head  (essential)
2. Print head pressure spring (essential, the one we tried did not work)
3. Print head bearing shells  (or a thingverse print head replacement) (demi-essential, could be glued but it won't last)
4. A nozzle & carrier - Olsson or similar is a better bet than an original. (non essentiasl as the nozzle could be unblocked, but **highly** recommended while it's apart.)

I have identified some contenders and left the info with Adrian.

**Comment by amcewen** [comment]
Thanks for the investigations, and hunting down some parts @Sean-anotherone.

Transferring the links you found to here so I can clear the paper off my desk :-D

 * [Block and nozzle](https://www.3djake.uk/3d-solex/matchless-nano-kit) at £83.40
 * [Steel coupler](https://www.3djake.uk/3d-solex/steel-coupler-v2-um2-) at £22.65

And then, if I'm right, it's a choice of...
 * 3D printing out [a new head](https://www.thingiverse.com/thing:945899) (in ABS so it copes best with the heat)
 * buying [a new head](https://www.makershop3d.com/ultimaker-spare-parts/681-um2-ext-go-plastic-print-head.html)

Given that we've already replaced part of one of the heads, that more-solid 3D printed design looks like a better option to me.

Do we still need to source a stronger spring for it all when we get to the reassembling part?

**Comment by amcewen** [comment]
I'd missed @Sean-anotherone's mention of the pressure spring above, so we _do_ still need a new spring.

It seems some people - e.g. https://www.youmagine.com/designs/um2-spring-replacement or https://www.youmagine.com/designs/ultimaker-2-spring-replacement use a spacer instead.

Having a look round for springs, it is available as part of [a hot end pack](https://3dgbire.com/collections/spare-parts/products/hot-end-pack-ultimaker-2-series?variant=1079476233).  That has a heater we don't want, but also a spare PTFE coupler (which we'd need eventually) and a coupler and block and nozzle.  It isn't as good as the nozzle replacement @Sean-anotherone suggested (or an Olssen block), but is only £75.60 all in (so ~£30 cheaper).

I think maybe we go for the cheaper option then, which would get us back to the same spec Felicity was before she broke.

(I also wonder if we should get a new [bowden tube and collets](https://3dgbire.com/collections/spare-parts/products/bowden-tube) at the same time as at least one of them isn't in great shape - especially the collets.  If we throw in 2 of [these timing belts](https://3dgbire.com/products/timing-belt-gt2-200?view=quickview) or up to 4 of [these longer timing belts](https://3dgbire.com/collections/spare-parts/products/timing-belt-gt2-610) that'd take us over £100 and get free shipping, so the belts would be free)

**Comment by jackie1050** [comment]
I've looked through this and agree this looks like it would fix the issue and give us some useful spares. It would be good to have 2 working printers especially if anyone wants to work in ABS.

**Comment by jackie1050** [comment]
I've ordered the parts in Adrian's last comment

**Comment by amcewen** [comment]
The parts @jackie1050 ordered seem to have arrived, and are next to Felicity awaiting fitting.

We still need to 3D print the [replacement head](https://www.thingiverse.com/thing:945899)

**Comment by Sean-anotherone** [comment]
I printed the new head at home and have reassembled Felicity.  I left a job running to see if the repair stands up.

The troublesome print ( I think several folks tried and failed) was solved by :

1. Trying twice and failing !
2. reslicing the model and changing to 250° extrusion, 75° bed and 65% printing speed, adding a full brim instead of a skirt.  There was something about the file as downloaded that caused the job to get snapped off the bed at around 75% complete, printing hotter and slower seemed to sort it out.  

I probably won't make it in this Thursday, but will bring Matt's roll of white ABS back as soon as I remember.
